aidb/inference/bound_inference_service.py

```diff
@@ -204,12 +210,15 @@ async def infer(self, inputs: pd.DataFrame):
     for idx, col in enumerate(self.binding.input_columns):
       inputs.rename(columns={inputs.columns[idx]: col}, inplace=True)
 
+    # Drop duplicate inputs
+    inputs_drop_duplicates = inputs.drop_duplicates()
```

Comment:
How does this map back to the output? If the call requests records [1, 2, 1], will it return [1, 2, 1]?

Reply:
The current 'infer' function is designed not to return results directly. Instead, results are obtained through an external SQL query, ensuring that this modification does not affect the results. 
In another PR, I have revised the 'infer' function to directly return results. This is implemented by joining the input and output on common columns, thereby aligning the output size with the input size.